crates/eips/src/eip2718.rs

```diff
@@ -29,7 +29,7 @@ pub const EIP7702_TX_TYPE_ID: u8 = 4;
 /// [EIP-2718] decoding errors.
 ///
 /// [EIP-2718]: https://eips.ethereum.org/EIPS/eip-2718
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
```

Comment:
what's the motivation for this?

hesitant to lock in support for PartialEq Eq in here in case we need to add an additional variant